components/card/index.css

```diff
@@ -745,8 +760,11 @@ governing permissions and limitations under the License.
 		.spectrum-Card-preview {
 			transition: none;
 			background-color: var(
-				--mod-card-background-color,
-				var(--spectrum-card-background-color)
+				--mod-card-preview-background-color, 
+				var(
+					--mod-card-background-color, 
```

Comment:
Should I mark `--mod-card-background-color` as being deprecated here? It was setting the background on the preview elements, while the main card background is set to transparent directly.

Reply:
I kinda like this hierarchy, reminds me of a font stack. Most specific is a preview-namespaced mod, then the general card mod, then the variable. So I don't mind it as is, but there's also not really a precedent for it in our components as far as I know. Not really an answer, just thoughts 😁

Reply:
What I was thinking is that ideally it seems like the gallery and quiet variations should be setting `--mod-card-background-color: transparent` in the CSS instead of `background-color: transparent;`, and then the `--mod-card-preview-background-color` would be used to set the background color. That seems to fit with our existing conventions and keeps the general bg color mod available for gallery. That would break any existing usage of `--mod-card-background-color` though.